doc/recipe/preprocessor.rst

```diff
@@ -374,6 +374,15 @@ or alternatively:
             {'short_name': 'sftof', 'exp': 'piControl'}
             ]
 
+Additionally, it is possible to search across all ensembles and experiments when specifying
```

Comment:
In theory, the wildcards also work for all other keys (not only `ensemble` and experiment`). This could be mentioned as a side note here.